plugins/woocommerce-admin/client/products/fills/attributes-section/attributes-section-fills.tsx

```diff
@@ -60,17 +59,11 @@ const AttributesSection = () => (
 			</ProductSectionLayout>
 		</WooProductSectionItem>
 		<WooProductFieldItem
-			id="attributes/add"
+			id={ `${ ATTRIBUTES_SECTION_ID }/add` }
 			sections={ [ { name: ATTRIBUTES_SECTION_ID, order: 1 } ] }
 			pluginId={ PLUGIN_ID }
 		>
 			<AttributesField />
 		</WooProductFieldItem>
 	</>
 );
-
-registerPlugin( 'wc-admin-product-editor-attributes-section', {
-	// @ts-expect-error 'scope' does exist. @types/wordpress__plugins is outdated.
-	scope: 'woocommerce-product-editor',
-	render: () => <AttributesSection />,
-} );
```

Comment:
Nice! Thanks for adding this all to a single `registerPlugin` call, I think that makes more sense.